build.gradle

```diff
@@ -18,12 +18,11 @@ ext {
     opensaml_version = '3.4.0'
     dropwizard_version = '1.3.9'
     utils_version = '2.0.0-350'
-    saml_libs_version = "${opensaml_version}-176"
+    saml_libs_version = "${opensaml_version}-179"
```

Comment:
Think the version is actually 180 for saml-libs, a dropwizard dependency change created 179

Reply:
updated